src/symbolic/symbolic_wasm_ffi.ml

```diff
@@ -69,6 +69,32 @@ module M :
       Fmt.pr "%c@?" (char_of_int (Int32.to_int c));
       Choice.return ()
     | _ -> assert false
+
+  let label m (id : Value.int32) (str_ptr : Value.int32) =
+    let open Choice in
+    let rec make_str accu i =
+      let* p = Memory.load_8_u m (Expr.value (Num (I32 i))) in
+      match Smtml.Expr.view p with
+      | Val (Num (I32 c)) ->
+        let ch = char_of_int @@ Int32.to_int c in
```

Comment:
`char_of_int` can fail:

> Raises Invalid_argument if the argument is outside the range 0--255.

I have not been able to hide it in `Prelude` because there is no safe equivalent in the stdlib (and I don't want to add new functions in prelude, only hide existing stuff).

It would be better to check the value of `Int32.to_int c` before calling it (we can simply `assert false` for now if it is not valid).